doc/release/platforms/README.cray

```diff
@@ -157,14 +178,38 @@ Using Chapel on a Cray System
           source util/setchplenv.bash
 
 
-2) Compile your Chapel program.  For example:
+2) Optionally, set one or more of the following environment variables to
+   select a Chapel configuration.  These are described in greater detail
+   in README.chplenv.
+
+     CHPL_LOCALE_MODEL : desired locale model, default "flat"
```

Comment:
Do you think it's worth mentioning the locale model? It seems like we could wait to include that here until we give it more attention WRT to performance